### PR TITLE
Update project website

### PR DESCRIPTION
**Pull request by lily983**
- Add features of PRIMP
- Add Delaware for Prof
- Upload video to YouTube(doing by Meng Xin)

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+- Change of viewing frame
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/dca9589d-1cbc-487c-9779-e7037fe1449d" alt="view frame" width="400" />
+</p>
+
+
+### Robot-agnostic
+- Skills can be easily transferred to another robot
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="robot agnostic" width="400" />
```
Comment (ruansp) [inline index.md]: The same figure with adaptation to via points?

File: index.md [hunk 2]
```diff
@@ -1,21 +1,54 @@
...
+</p>
+
+### Adaptation to unseen scenarios
+- Avoid novel obstacles while keep the motion features
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="avoid obs" width="400" />
```
Comment (ruansp) [inline index.md]: Also the same figure?

File: index.md [hunk 3]
```diff
@@ -1,21 +1,54 @@
...
+
+
+### Robot-agnostic
+- Skills can be easily transferred to another robot
+<p align="center">
+<img src="https://github.com/ChirikjianLab/primp-page/assets/71541515/49382079-edd6-4c35-9731-4451c08c3799" alt="robot agnostic" width="400" />
+</p>
+
+### Adaptation to unseen scenarios
+- Avoid novel obstacles while keep the motion features
```
Comment (ruansp) [inline index.md]: -> "Avoid previously unseen obstacles while keeping the key features"

**Review by ruansp — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (lily983) [reply index.md]: OK. I have changed the link, but I think I didn't successfully save them. I will update figure links.

**Comment by lily983** [comment]
Hi Sipu, I have fixed the figure link errors and added more explanation to sub-titles. BTW, do we need to update the paper on Arxiv? The current version is not up-to-date.

**Comment by ruansp** [comment]
> Hi Sipu, I have fixed the figure link errors and added more explanation to sub-titles. BTW, do we need to update the paper on Arxiv? The current version is not up-to-date.

I don't think it is necessary since our paper in IEEE Xplore is open access. Also better not to be the same with the published version, which might cause copyright issues. I have linked the journal reference using DOI.

**Review by ruansp — APPROVED** [review]

**Review by ruansp — APPROVED** [review]

**Pull request merged**